torch/_inductor/template_heuristics/triton.py

```diff
@@ -1858,6 +1858,27 @@ def _get_template_configs_impl(
             yield template_kwargs
 
 
+# Scaled Blackwell TMA-specific mixin for scaled MM templates with TMA
+class ScaledBlackwellTMAConfigMixin(BlackwellTMATemplateConfigMixin, ScaledMMConfigMixin):
+    """
+    Scaled Blackwell TMA-specific mixin that extends ScaledMMConfigMixin with TMA functionality.
+    This is for scaled MM templates that use device TMA on Blackwell.
+    This inherits from ScaledMMConfigMixin, which inherits the scale_mm_epilogue, and adds TMA-specific options.
+    """
+
+    def _filter_configs(self, configs: list[BaseConfig]) -> list[BaseConfig]:
+        """
+        Warp specialization-specific filtering (BlackwellTMATemplateConfigMixin):
+        - num_warps < 4 not safe for warpspec due to compilation issues
```

Comment:
Nit: This is only in some versions of Triton. I might add that clarification in case someone tries to remove it because it works for them upstream.